packages/paste-website/src/pages/patterns/filter/index.mdx

```diff
@@ -0,0 +1,698 @@
+
+## Ingredients
+
+<Grid gutter="space30" marginBottom="space80">
+  <Column>
+    <Card>
+      <Heading as="h3" variant="heading40" marginBottom="space0">
+        <Anchor href="/components/form-pill-group">Form Pill Group</Anchor>
+      </Heading>
+    </Card>
+  </Column>
+   <Column>
+    <Card>
+      <Heading as="h3" variant="heading40" marginBottom="space0">
+        <Anchor href="/components/input">Input</Anchor>
+      </Heading>
+    </Card>
+  </Column>
+  <Column>
+    <Card>
+      <Heading as="h3" variant="heading40" marginBottom="space0">
+        <Anchor href="/components/multiselect-combobox">Multiselect Combobox</Anchor>
+      </Heading>
+    </Card>
+  </Column>
+</Grid>
+<Grid gutter="space30" marginBottom="space80">
+  <Column>
+    <Card>
+      <Heading as="h3" variant="heading40" marginBottom="space0">
+        <Anchor href="/components/button">Button</Anchor>
+      </Heading>
+    </Card>
+  </Column>
+  <Column>
+    <Card>
+      <Heading as="h3" variant="heading40" marginBottom="space0">
+        <Anchor href="/components/button-group">Button Group</Anchor>
+      </Heading>
+    </Card>
+  </Column>
+  <Column>
+    <Card>
+      <Heading as="h3" variant="heading40" marginBottom="space0">
+        <Anchor href="/components/disclosure">Disclosure</Anchor>
+      </Heading>
+    </Card>
+  </Column>
+</Grid>
+<Grid gutter="space30" marginBottom="space80">
+  <Column>
+    <Card>
+      <Heading as="h3" variant="heading40" marginBottom="space0">
+        <Anchor href="/components/badge">Badge</Anchor>
+      </Heading>
+    </Card>
+  </Column>
+  <Column>
+    <Card>
+      <Heading as="h3" variant="heading40" marginBottom="space0">
+        <Anchor href="/components/checkbox">Checkbox</Anchor>
+      </Heading>
+    </Card>
+  </Column>
+  <Column>
+    <Card>
+      <Heading as="h3" variant="heading40" marginBottom="space0">
+        <Anchor href="/components/radio-group">Radio</Anchor>
+      </Heading>
+    </Card>
+  </Column>
+   <Column>
+    <Card>
+      <Heading as="h3" variant="heading40" marginBottom="space0">
+        <Anchor href="/components/popover">Popover</Anchor>
+      </Heading>
+    </Card>
+  </Column>
+</Grid>
+
+```jsx
+// import the components for filter group patterns as required
+
+import { FormPillGroup, useFormPillState } from "@twilio-paste/form-pill-group";
+import {Input} from '@twilio-paste/input';
+import {MultiselectCombobox, useMultiselectCombobox} from "@twilio-paste/combobox";
+import {Button} from '@twilio-paste/button';
+import {ButtonGroup} from "@twilio-paste/button-group";
+import {Disclosure, DisclosureContent, DisclosureHeading} from "@twilio-paste/disclosure";
+import {Badge} from "@twilio-paste/badge";
+import {Checkbox, CheckboxGroup} from "@twilio-paste/checkbox";
+import {Radio, RadioGroup} from "@twilio-paste/radio-group";
+import { Popover, PopoverButton, PopoverContainer, usePopoverState } from "@twilio-paste/popover";
```

Comment:
blocking: Consumers should always import the components from core:

import { FormPillGroup, useFormPillState } from "@twilio-paste/core/form-pill-group";